.github/workflows/test.yaml

```diff
@@ -21,7 +21,7 @@ jobs:
       fail-fast: false
       matrix:
         os: [ubuntu-latest, windows-latest, macos-latest]
-        python-version: ['3.6', '3.10']
+        python-version: ['3.7', '3.10']
```

Comment:
Playwright needs Python>=3.7

Reply:
While Python 3.6 just hit EoL, since we've preserved support up to the docrepr 0.2.0 release, I'd think it makes sense to release at least one version with it. Given it would be helpful to have the status of the playwright checks reported separately from the rest, and inter-Python-version differences that cause meaningful deltas in the output but don't fail any previous checks or raise an error (unsupported syntax, etc) seem very unlikely, I'd suggest just including a separate set of jobs in the matrix, one on each platform using e.g. Py3.8 that passes the custom pytest flag to enable the screenshot checks.